torch_frame/data/multi_embedding_tensor.py

```diff
@@ -0,0 +1,166 @@
+import copy
+from typing import Any, Callable, Sequence, Tuple
+
+import torch
+from torch import Tensor
+
+
+class MultiEmbeddingTensor:
+    r"""A PyTorch tensor-based data structure that stores
+    :obj:`[num_rows, num_cols, *]`, where the size of last dimension can be
```

Comment:
QQ: the dimension for the same column should be same right?

Reply:
Yes. I think it would be great to highlight in the doc what's the difference between `MultiNestedTensor` and `MultiEmbeddingTensor`.

Reply:
Briefly described the difference between these two. PTAL :)